libpod/container_internal_linux.go

```diff
@@ -1365,7 +1365,7 @@ func (c *Container) getHosts() string {
 }
 
 // generatePasswd generates a container specific passwd file,
-// iff g.config.User is a number
+// if g.config.User is a number
```

Comment:
This is definitely wrong - iff is accepted terminology

Reply:
Fixed